demisto_sdk/commands/init/tests/contribution_converter_test.py

```diff
@@ -101,21 +177,39 @@ def test_convert_contribution_zip(get_content_path_mock, get_python_version_mock
     sample_script_path = scripts_path / 'SampleScript'
     script_yml = sample_script_path / 'SampleScript.yml'
     script_py = sample_script_path / 'SampleScript.py'
+    script_readme_md = sample_script_path / 'README.md'
+    unified_script = sample_script_path / 'script-script0.yml'
```

Comment:
Also please check the unified doesn't exist under the `Scripts` directory, similar to what we do below with the possible paths for the unified integration file.